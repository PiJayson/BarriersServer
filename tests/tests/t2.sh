#!/bin/sh

. common

$BIN/errno.x 

