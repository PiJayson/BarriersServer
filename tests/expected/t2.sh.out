Reinitializaion test:
Init barrier id: 19 width: 2 ... result 0
Init barrier id: 19 width: 2 ... result -1 errno: 17
...done.

Destroying unitialized:
Destroying barrier id: 19 ... result 0
Destroying barrier id: 19 ... result -1 errno:22
...done.

Waiting on unitialized:
...done.

Destroying barrier that is being used:
Destroying barrier id: 19 ... result -1 errno:16
Wait result: 0, errno 16
...done.
